jekyll/_cci2/api-developers-guide.md

```diff
@@ -348,3 +348,73 @@ job -
 
 
 ## Download Artifacts
+
+The following section details the steps you need to follow to download artifacts generated when a job is run. We will go through the steps of returning a list of artifacts for a job, and then changing the command we run to download the full set of artifacts. If you are looing for instructions to download the latest artifacts for a pipeline, without needing to specify a job number, see our [API v1.1 guide](https://circleci.com/docs/2.0/artifacts/#downloading-all-artifacts-for-a-build-on-circleci) – keep checking back here as this functionality will be added to API v2 in the future.
+
+### Prerequisites
+
+Before making an API call, make sure you have met the following prerequisites:
+
+* You have set up a GitHub or BitBucket account with a repository to use with CircleCI.
+* You have completed CircleCI onboarding and you have a project setup.
+* You have a [personal API token](https://circleci.com/account/api).
+* You have been [authenticated](#get-authenticated) to make API calls to the server.
+
+### Steps
+
+1. First we will ensure your api token is set as an environment variable. You maybe have already done this during authentication, but if not, run the following substituting your personal api token:
+
+    ```
+    export CIRCLECI_TOKEN={your_api_token}
+    ```
+
+2.  Next we need the job number for the job you want to get artifacts for. You can find job numbers in the UI - either in the breadcrumbs on the Job Details page, or in the URL.
+
+    INSERT SCREENSHOT HIGHLIGHTING JOB NUMBER
+
+3.  Next we can use `curl` to return a list of artifacts for a specific job. Note, you can format JSON responses by piping the `curl` command into the `jq` utility if you have it installed: `curl ... | jq`:
+
+    ```sh
+    curl -X GET https://circleci.com/api/v2/project/{VCS}/{org-name}/{repo-name}/{job-number}/artifacts \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLECI_TOKEN" | jq\
+    ```
+
+    Check the table below for help with formatting your API call correctly
+
+    Parameter | Description
+    --- | ---
+    `VCS` | Version Control System - either `github`/`gh` or `bitbucket`/`bb`
+    `org-name` | Organization name, or your personal username to your VCS
+    `repo-name` | Name of your project repo
+    `job-number` | The number for the job you want to download artifacts from - see step 2
+    
+    <BR>
```

Comment:
We should avoid using html in markdown -- if this is a spacing issue we might be able to solve it a different way.

Reply:
@teesloane Yeah it looked weird due to the text starting straight after the table. I just stuck this in as a quick fix but I'm wondering why we should avoid HTML in cases like this?

Reply:
Generally, it's more that `<BR>` is not an ideal html tag -- it's not [semantic](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br) - although you could argue I'm being _pedantic_ ;)

Regardless, it looks like there are only [three ways](https://stackoverflow.com/a/48583859) to create blank lines/empty spaces in markdown — so feel free to go with the `<br/>`? One other alternative I can think of would be to maybe try inserting an `<hr />`  by using `***` or `---` which is semantic markdown.

Reply:
AH ok! Yeah when I stuck <BR> in it was the result of the first stack overflow hit I got for how to add line space in markdown. Will check the other options.

Reply:
Yeah I tried --- but then I get a horizontal rule so not sure. I will format the table correctly and maybe that will solve my problem anyway

Reply:
That worked!